test/test_mps.py

```diff
@@ -57,6 +58,47 @@
     )
 )
 
+def mps_ops_modifier(ops):
+    # Those ops worked on MacOS12, but broken on MacOS13, see https://github.com/pytorch/pytorch/issues/85758
+    VENTURA_XFAILLIST = {
```

Comment:
nit: since I feel like we're going to need variants of this for all minor versions, we might not want to use the name here but the version directly?

Reply:
Makes sense. We can use : `MACOS_12_3, MACOS_13_0, MACOS_13_1` ...

Reply:
Added MACOS_13_X_ and MACOS_12_X_